<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeConfig</class>
 <widget class="QDialog" name="NodeConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="cbType">
     <item>
      <property name="text">
       <string>Geometry</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Billboards</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="sbSize">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Geometry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbSubdiv">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Subdivisions</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="cbSmooth">
        <property name="text">
         <string>Smooth</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NodeConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NodeConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbSize</sender>
   <signal>valueChanged(double)</signal>
   <receiver>NodeConfig</receiver>
   <slot>OnSizeChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbSubdiv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NodeConfig</receiver>
   <slot>OnSubdivisionsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NodeConfig</receiver>
   <slot>OnTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSmooth</sender>
   <signal>stateChanged(int)</signal>
   <receiver>NodeConfig</receiver>
   <slot>OnSmoothChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSizeChanged(double)</slot>
  <slot>OnSubdivisionsChanged(int)</slot>
  <slot>OnTypeChanged(int)</slot>
  <slot>OnSmoothChanged(int)</slot>
 </slots>
</ui>
